src/Doctrine/ObjectManagerAwareTrait.php

```diff
@@ -2,27 +2,27 @@
 
 namespace Detail\Persistence\Doctrine;
 
-use Doctrine\Common\Persistence\ObjectManager as ObjectManagerInterface;
+use Doctrine\Persistence\ObjectManager;
 
 trait ObjectManagerAwareTrait
 {
     /**
-     * @var ObjectManagerInterface
+     * @var ObjectManager
      */
```

Comment:
Use one-liner

Reply:
I don't want to mix code style improvements together with this change.